Samples/Shapes/Scripts/ColorChanger.cs

```diff
@@ -27,18 +27,18 @@ public void UpdateColor(WitResponseNode response)
 
                 if (ColorUtility.TryParseHtmlString(colorString, out var color))
                 {
-                    if (!string.IsNullOrEmpty(shapeString))
-                    {
-                        var shape = transform.Find(shapeString);
-                        if(shape) SetColor(shape, color);
-                    }
-                    else
+                    if (string.IsNullOrEmpty(shapeString))
```

Comment:
Would this code block mean that "make the bla red" would make all shapes red, if bla failed to resolve?

Reply:
Yes